runtime/doc/autocmd.txt

```diff
@@ -994,7 +994,7 @@ expect.
 Environment variables can be used in a pattern: >
 	:autocmd BufRead $VIMRUNTIME/doc/*.txt  set expandtab
 And ~ can be used for the home directory (if $HOME is defined): >
-	:autocmd BufWritePost ~/.vimrc   so ~/.vimrc
+	:autocmd BufWritePost ~/.config/nvim/init.vim   so <afile>
```

Comment:
> I guess we need another section somewhere in documentation (options.txt?) with explanations.
I think that would be a good idea. It's not too complicated, so I think it should be doable in a paragraph or two, at least as a high level overview.